llvm/test/CodeGen/M68k/Data/load-extend.ll

```diff
@@ -41,3 +41,48 @@ define i32 @"test_zext_pcd_i16_to_i32"() {
   %val2 = zext i16 %val to i32
   ret i32 %val2
 }
+
+define i16 @"test_anyext_pcd_i8_to_i16"() {
```

Comment:
I know you copied from tests above, but could you (1) use normal function name without double quotes like `@test_anyext_pcd_i8_to_i16` (2) add `nounwind` so that with can get rid of the CFI directives like `.cfi_startproc`.

Reply:
did both of these